api/stream-portfolio/openapi.yaml

```diff
@@ -44,6 +47,37 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  ###############################################
+  ## Integration
+  ###############################################
+  /integration-api/v1/regions/batch:
```

Comment:
I would prefer to this api keep consistency with the existing endpoints, please change it to `/regions/batch`

Reply:
or maybe `/portfolios/regions/batch` if makes more sense

Reply:
Done